core/ui/menu/settings/plugin_store_menu.gd

```diff
@@ -67,4 +69,14 @@ func _populate_plugin_store_items(grid: Container, plugin_items: Dictionary):
 		
 		# Add the store item
 		store_item.visible = true
-		
+
+
+func _on_plugin_upgradable(plugin_id: String, update_type: int) -> void:
+	var notify := Notification.new("")
+	if update_type == PluginLoader.update_type.NEW:
+		notify.text=("New plugin available: {0}".format([plugin_id]))
+
+	if update_type == PluginLoader.update_type.UPDATE:
+		notify.text=("Plugin upgrade available: {0}".format([plugin_id]))
+
+	NotificationManager.show(notify)
```

Comment:
Doing this every 5 minutes until they update seems annoying. Can we set a flag somewhere that they've already been notified for this plugin id?

Reply:
It wont notify every five minutes, it will only check for additional changes in the store each time. we save the most current database to check against if it was different from the last run. See line 486.

Reply:
I added a check just in case. https://github.com/ShadowBlip/OpenGamepadUI/pull/55/commits/6e3999147ebe49679809e52e4eeb3d494c826477